experiments/standalone/Bucket/global_bucket.jl

```diff
@@ -0,0 +1,200 @@
+    TimeVaryingInput(LW_d),
+    ref_time,
+);
+
+model = BucketModel(
+    parameters = bucket_parameters,
+    domain = bucket_domain,
+    atmosphere = bucket_atmos,
+    radiation = bucket_rad,
+);
+
+Y, p, coords = initialize(model);
+
+Y.bucket.T .= FT(270);
+Y.bucket.W .= FT(0.05);
+Y.bucket.Ws .= FT(0.0);
+Y.bucket.σS .= FT(0.08);
+
+set_initial_cache! = make_set_initial_cache(model);
+set_initial_cache!(p, Y, t0);
+exp_tendency! = make_exp_tendency(model);
+timestepper = CTS.RK4()
+ode_algo = CTS.ExplicitAlgorithm(timestepper)
+prob = SciMLBase.ODEProblem(
+    CTS.ClimaODEFunction(T_exp! = exp_tendency!, dss! = ClimaLSM.dss!),
+    Y,
+    (t0, tf),
+    p,
+);
+
+saveat = collect(t0:(Δt * 3):tf);
+saved_values = (;
+    t = Array{Float64}(undef, length(saveat)),
+    saveval = Array{NamedTuple}(undef, length(saveat)),
+);
+saving_cb = ClimaLSM.NonInterpSavingCallback(saved_values, saveat);
+updateat = deepcopy(saveat)
+updatefunc = ClimaLSM.make_update_drivers(bucket_atmos, bucket_rad)
+driver_cb = ClimaLSM.DriverUpdateCallback(updateat, updatefunc)
+cb = SciMLBase.CallbackSet(driver_cb, saving_cb)
+
+sol = SciMLBase.solve(prob, ode_algo; dt = Δt, saveat = saveat, callback = cb);
+
+# Interpolate to grid
+space = axes(coords.surface)
+longpts = range(-180.0, 180.0, 21)
+latpts = range(-90.0, 90.0, 21)
+hcoords = [Geometry.LatLongPoint(lat, long) for long in longpts, lat in latpts]
+remapper = Remapping.Remapper(hcoords, nothing, space)
+
+W = [
+    Remapping.interpolate(remapper, sol.u[k].bucket.W) for k in 1:length(sol.t)
+];
+σS = [
+    Remapping.interpolate(remapper, sol.u[k].bucket.σS) for k in 1:length(sol.t)
+];
+T_sfc = [
+    Remapping.interpolate(remapper, saved_values.saveval[k].bucket.T_sfc)
+    for k in 1:length(sol.t)
+];
+evaporation = [
+    Remapping.interpolate(
+        remapper,
+        saved_values.saveval[k].bucket.turbulent_fluxes.vapor_flux,
+    ) for k in 1:length(sol.t)
+];
+F_sfc = [
+    Remapping.interpolate(
+        remapper,
+        saved_values.saveval[k].bucket.R_n .+
+        saved_values.saveval[k].bucket.turbulent_fluxes.lhf .+
+        saved_values.saveval[k].bucket.turbulent_fluxes.shf,
+    ) for k in 1:length(sol.t)
+];
+# animation settings
+nframes = length(W)
+framerate = 2
+
+for (field_ts, field_name) in zip(
+    [W, σS, T_sfc, evaporation, F_sfc],
+    ["W", "σS", "T_sfc", "evaporation", "F_sfc"],
+)
+    fig = Figure(size = (1000, 1000))
+    ax = Axis(
+        fig[1, 1],
+        xlabel = "Longitude",
+        ylabel = "Latitude",
+        title = field_name,
+    )
+    clims = compute_extrema(field_ts)
+    CairoMakie.Colorbar(fig[:, end + 1], colorrange = clims)
+    outfile = joinpath(outdir, string("anim_", field_name, ".mp4"))
+    record(fig, outfile, 1:nframes; framerate = framerate) do frame
+        CairoMakie.heatmap!(
+            longpts,
+            latpts,
+            field_ts[frame],
+            colorrange = clims,
+        )
+    end
+end
```

Comment:
Atmos found animations to be difficult to inspect and debug, maybe we can also make some of the frames accessible?

Reply:
good call. they really are hard to inspect. In this case the output is extremely boring (constant everywhere, no lat/lon dependence in forcing), so I will also make timeseries of the (single valued except for roundoff) value of the state vars as a function of time.

Reply:
and turn off the anims for now